### PR TITLE
Pa11y dashboard

### PR DESCRIPTION
**Pull request by gestchild**
The [deploy of the pa11y report is failing](https://buildkite.com/wellcomecollection/wc-dot-org-deployment/builds/1622#01846100-a3be-40a8-9c46-eb86ac8a0633) because the generation of the report is timing out. I couldn't see any obvious reason why, so have increased the timeout, which will hopefully fix the problem.

I've also added another page to test and prevented some warnings about max listeners and uncaught errors

### CODE REVIEW

File: pa11y/webapp/write-report.js [hunk 1]
```diff
@@ -30,6 +33,7 @@ const urls = [
   '/events/Wqkd1yUAAB8sW4By',
   '/event-series/WlYT_SQAACcAWccj',
   '/concepts/n4fvtc49',
+  '/guides/YL9OAxIAAB8AHsyv',
```
Comment (rcantin-w) [inline pa11y/webapp/write-report.js:36]: I think it needs to be added to the URL checker as well (if it's new?)

>// Note: if you add a URL to this list, make sure to also add it to the list
// of URLs checked by the URL checker.
//
// See .buildkite/urls/expected_200_urls.txt
//

**Review by rcantin-w — APPROVED** [review]
🤞 👀

File: pa11y/webapp/write-report.js (hunk 1 above)
Comment (gestchild) [reply pa11y/webapp/write-report.js:36]: good point, thanks

Comment (rcantin-w) [reply pa11y/webapp/write-report.js:36]: So it is! I know Alex started [some work on that](https://github.com/wellcomecollection/wellcomecollection.org/pull/8817) and I think I thought they were already aligned. All good!

**Pull request merged**